docs/source/about/guides/app_examples.rst

```diff
@@ -256,6 +256,22 @@ Changing Resolution and Refresh Rate
       |                      | Undo: ``wlr-xrandr --output HDMI-1 --mode 3840x2160@120Hz``                                                                       |
       +----------------------+-----------------------------------------------------------------------------------------------------------------------------------+
 
+      .. hint::
+
+         ``wlr-xrandr`` only works with wlroots-basec compositors.
+
+   .. tab:: Gnome (Wayland, X11)
+
+      +----------------------+----------------------------------------------------------------------------------------------------------------------------------------+
+      | Command Preparations | Do: ``sh -c "gnome-randr.py --output HDMI-1 --mode ${SUNSHINE_CLIENT_WIDTH}x${SUNSHINE_CLIENT_HEIGHT} --rate ${SUNSHINE_CLIENT_FPS}"`` |
+      |                      +----------------------------------------------------------------------------------------------------------------------------------------+
+      |                      | Undo: ``gnome-randr.py --output HDMI-1 --mode 3840x2160 --rate 120``                                                                   |
+      +----------------------+----------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
Without the `sh -c` there's nothing that does the expansion of the variables to their actual values. We could probably modify Sunshine to always execute those Do/Undo commands via the shell. It would make the syntax more straightforward. But currently the `sh -c` is necessary.

Reply:
Okay, that makes sense.